timeless/items/models.py

```diff
@@ -29,16 +29,19 @@ def __init__(self, **kwargs):
     def assign(self, employee):
         """ Assigning the item to an employee """
         self.employee_id = employee.id
-        item_history = ItemHistory(employee_id=self.employee_id, item_id=self.id)
+        self.history.append(
+            ItemHistory(
+                employee_id=self.employee_id,
+                item_id=self.id,
+                start_time=datetime.utcnow
+            )
```

Comment:
@paulodamaso before simply appending the new ``ItemHistory``, maybe we can iterate over them, find the latest with ``item_id == self.id`` and set its ``end_time`` (that is, the time when the Item was handed over from one employee to the next. Makes sense?